backend/src/jwt/decode.js

```diff
@@ -14,6 +14,7 @@ export default async (driver, authorizationHeader) => {
   const session = driver.session()
   const query = `
     MATCH (user:User {id: $id, deleted: false, disabled: false })
+    SET user.lastActiveAt = datetime()
```

Comment:
Actually, can you convert this `datetime()` to a string, at the moment we are doing this cause of a bug with one of the packages we are using for queries 
```suggestion
    SET user.lastActiveAt = toString(datetime())
```